src/Illuminate/Foundation/Vite.php

```diff
@@ -397,4 +419,63 @@ public function reactRefresh()
             )
         );
     }
+
+    /**
+     * Merge the given entry points.
+     *
+     * @param  array  $entryPoints
+     * @return $this
+     */
+    public function withEntryPoints(array $entryPoints)
+    {
+        $this->entryPoints = array_merge($this->entryPoints, $entryPoints);
+
+        return $this;
+    }
+
+    /**
+     * Set the hot file path attribute.
+     *
+     * @param  string
+     * @return $this
+     */
+    public function useHotFile(string $path)
+    {
+        $this->hotFilePath = $path;
+
+        return $this;
+    }
+
+    /**
+     * Get the hot file path.
+     *
+     * @return string
+     */
+    protected function hotFilePath()
+    {
+        return $this->hotFilePath ?? public_path('/hot');
```

Comment:
@timacdonald I think we should have something like this for the `buildDirectory` as well. But it does not seem to be that simple. What do you think?

Reply:
Right now it's not very consistent because the `buildDirectory` path we can pass to the `__invoke` and the `useBuildDirectory` methods can be relative only.

Reply:
Actually this should be fine, since the `build` directory **must be** in the public directory.